avalanche/training/plugins/from_scratch_training.py

```diff
@@ -0,0 +1,54 @@
+from copy import deepcopy
+
+from avalanche.training.plugins.strategy_plugin import SupervisedPlugin
+from avalanche.training.templates import SupervisedTemplate
+from avalanche.models.dynamic_optimizers import reset_optimizer
+
+
+class FromScratchTrainingPlugin(SupervisedPlugin):
```

Comment:
I think it's better to use the `SGDTemplate` here to support more use cases.

Reply:
Agreed, will change it!